### ISSUE TITLE
"help/info" page from within form

### ISSUE THREAD

**Issue by MikeKutz** [oddgen/oddgen#30]
Within the "Generate..." form, the ability to view a custom "help/info" page would be nice.

Example operation:
If a "get_help()" function exists within the generator, an icon for "help" would appear in the top right corner of the form.

Clicking on the "help" icon would open a floatable window that displays the results of the "get_help()" function.  That is, you can read the "help/info" documentation while filling out the form.

The "get_help" function should return a CLOB of some pre-established markup language.

eg

``` sql
subtype HTML clob;
function get_help return HTML;
```

MK

**Comment by PhilippSalvisberg** [comment]
Good idea. I add that to v0.3.0 milestone, since we are amending the interfaces once again anyway. ;-)
